fonduer/candidates/models/candidate.py

```diff
@@ -207,17 +208,31 @@ class Marginal(_meta.Base):
 
     @training: If True, this is a training marginal; otherwise is end prediction
     """
-    __tablename__ = 'marginal'
+
+    __tablename__ = "marginal"
     id = Column(Integer, primary_key=True)
-    candidate_id = Column(Integer,
-                          ForeignKey('candidate.id', ondelete='CASCADE'))
+    candidate_id = Column(Integer, ForeignKey("candidate.id", ondelete="CASCADE"))
     training = Column(Boolean, default=True)
     value = Column(Integer, nullable=False, default=1)
     probability = Column(Float, nullable=False, default=0.0)
 
-    __table_args__ = (UniqueConstraint(candidate_id, training, value), )
+    __table_args__ = (UniqueConstraint(candidate_id, training, value),)
 
     def __repr__(self):
         label = "Training" if self.training else "Predicted"
-        return "<%s Marginal: P(%s == %s) = %s>" % \
-            (label, self.candidate_id, self.value, self.probability)
+        return "<%s Marginal: P(%s == %s) = %s>" % (
+            label,
+            self.candidate_id,
+            self.value,
+            self.probability,
+        )
+
+
+def init_models():
```

Comment:
What does this function do?

Reply:
Good catch, this was from when I was going to split the initialization. This should now be deleted.